jobs/kpi-forecasting/requirements.txt

```diff
@@ -1,117 +1,75 @@
-adagio==0.2.4
-ansi2html==1.8.0
-antlr4-python3-runtime==4.11.1
-appdirs==1.4.4
-asttokens==2.2.1
-backcall==0.2.0
+attrs==23.1.0
 black==23.3.0
-blinker==1.6.2
-cachetools==5.3.0
+cachetools==5.3.1
+cattrs==23.1.2
 certifi==2023.5.7
 charset-normalizer==3.1.0
 click==8.1.3
 cmdstanpy==1.1.0
-comm==0.1.3
-contourpy==1.0.7
+contourpy==1.1.0
 convertdate==2.4.0
 cycler==0.11.0
-dash==2.9.3
-dash-core-components==2.0.0
-dash-html-components==2.0.0
-dash-table==5.0.0
 db-dtypes==1.1.1
-debugpy==1.6.7
-decorator==5.1.1
+dotmap==1.3.30
 ephem==4.1.4
 exceptiongroup==1.1.1
-executing==1.2.0
-Flask==2.3.2
-fonttools==4.39.3
-fs==2.4.16
-fugue==0.8.3
-fugue-sql-antlr==0.1.6
-google-api-core==2.11.0
-google-auth==2.17.3
-google-cloud-bigquery==3.10.0
+fonttools==4.40.0
```

Comment:
Are all of these requirements necessary?

Reply:
As far as I know they are... I started from a fresh environment and only installed the packages required to get the code running.